opensfm/reconstruction.py

```diff
@@ -176,21 +177,36 @@ def pairwise_reconstructability(common_tracks, homography_inliers):
 
 def compute_image_pairs(graph, image_graph, config):
     """All matched image pairs sorted by reconstructability."""
-    pairs = []
-    score = []
-    for im1, im2, d in image_graph.edges(data=True):
-        tracks, p1, p2 = matching.common_tracks(graph, im1, im2)
-        if len(tracks) >= 50:
-            H, inliers = cv2.findHomography(
-                p1, p2, cv2.RANSAC, config.get('homography_threshold', 0.004))
-            r = pairwise_reconstructability(len(tracks), inliers.sum())
-            if r > 0:
-                pairs.append((im1, im2))
-                score.append(r)
+    args = _pair_reconstructability_arguments(graph, image_graph, config)
+    processes = config.get('processes', 1)
+    if processes == 1:
+        result = map(_compute_pair_reconstructability, args)
+    else:
+        p = Pool(processes)
+        result = p.map(_compute_pair_reconstructability, args)
+    pairs = [(im1, im2) for im1, im2, r in result if r > 0]
+    score = [r for im1, im2, r in result if r > 0]
     order = np.argsort(-np.array(score))
     return [pairs[o] for o in order]
 
 
+def _pair_reconstructability_arguments(graph, image_graph, config):
+    homography_threshold = config.get('homography_threshold', 0.004)
+    return [(homography_threshold, im1, im2, d, matching.common_tracks(graph, im1, im2)) 
```

Comment:
Would it work to pass `graph` as an argument instead of `matching.common_tracks(graph, im1, im2)` and then compute the common_tracks in `_compute_pair_reconstructability`?